src/Types/Unify.hs

```diff
@@ -460,15 +461,17 @@ bind scope var ty
   | TyVar var == ty = pure (ReflCo ty)
   -- /\ Var-var deletion
   | TyWildcard (Just (TyVar v)) <- ty, v == var = pure (ReflCo ty)
-  -- /\ Var-wildcard deletion (same as above, but with an indirection)
-  | TyPi (Invisible _ _ r) _ <- ty, r /= Req = confesses (Impredicative var ty)
   -- /\ Predicativity checking
   | otherwise = do
+      -- guarded <- view guarded
```

Comment:
Can we remove this (and the lines just below)?

Reply:
Yeah